students/carlos_novoa/lesson04/mailroom.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
+import os
+import sys
+import datetime
+
+"""
+Lesson4 - Mailroom, Part 2
+"""
+
+now = datetime.datetime.now()
+
+_donors = [
+    [
+        {
+            'first_name': 'Jim',
+            'last_name': 'Halpert'
+        },
+        {
+            'donations': [
+                1.00
+            ]
+        }
+    ],
```

Comment:
if you are to use a list then you would do a list of dictionaries, there is no reason to have two dicts, so it should look like this:
```
donors = [{"first_name": "Jim", "last_name": "Halpert", "donations": [1.00]}, {"first_name": "Pam", "last_name": "beesly", "donations": [3]}]
```